src/js/tabs/advanced.js

```diff
@@ -178,6 +178,8 @@ AdvancedTab.prototype.angular = function(module)
         if (!store.disabled) {
           store.set('ripple_settings', JSON.stringify($scope.options));
         }
+        var servers = settings.getClearServers($scope.options.server.servers);
```

Comment:
save and remove have 5 lines of the same code, can you move them to a separate method? let's say `.persist()`